src/common/libkvs/kvs_txn.c

```diff
@@ -53,7 +53,6 @@
  *
  * NULL or empty values:
  * A zero length raw value is considered valid.
- * A NULL JSON string passed to flux_kvs_txn_put() is interpreted as an unlink.
```

Comment:
Perhaps a comment would be useful stating that NULL is invalid, but if user wishes to put a zero length value, to use ```flux_kvs_txn_put_raw()```.

Reply:
Yes or actually it should be valid to put a NULL there.